docs/content/tendermint-and-cosmos.rst

```diff
@@ -0,0 +1,93 @@
+Tendermint & Cosmos
+===================
+
+We're building a lot of software under two different GitHub organizations:
```

Comment:
In general, since it's a documentation, I would refrain from contractions like "We're", "There's" etc.

Reply:
agree, this was copied over from elsewhere